packages/main/src/RadioButton.ts

```diff
@@ -121,6 +121,8 @@ class RadioButton extends UI5Element implements IFormInputElement {
 
 	/**
 	 * Defines whether the component is checked or not.
+	 * Only enabled radio buttons can be checked.
+	 * Read-only radio buttons are not selectable, and therefore are always unchecked.
```

Comment:
I suggest to reorder the notes:
Defines whether the component is checked or not.
**Note:** The property value can be changed with user interaction, either by clicking/tapping on the component, or by using the `Space` or `Enter` key.
**Note:** Only enabled radio buttons can be checked. Read-only radio buttons are not selectable, and therefore are always unchecked.